src/index.js

```diff
@@ -34,7 +34,7 @@ export function getSpecReferences() {
   };
 
   // Wrap jasmine's describe function to gather references to all suites.
-  jasmine.getEnv().describe = _.wrap(jasmine.getEnv().describe,
+  jasmine.getEnv().describe = wrap(jasmine.getEnv().describe,
     (describe, ...args) => {
       let suite = describe.apply(null, args);
       suites.push(suite);
```

Comment:
```suggestion
  const _describe = jasmine.getEnv().describe;
  jasmine.getEnv().describe = (...args) => {
      let suite = _describe.apply(null, args);
      suites.push(suite);
```

or could even take it one step further and remove lodash altogether 🤷‍♂️